app/scripts/controllers/transactions/lib/util.js

```diff
@@ -164,6 +164,11 @@ export function validateTxParams(txParams, eip1559Compatibility = true) {
           'maxPriorityFeePerGas',
         );
         ensureFieldIsString(txParams, 'gasPrice');
+        if (!value.match(/^0x[a-fA-F0-9]+$/u)) {
```

Comment:
Would it be possible and cleaner if you assign that regex to some variable instead of keeping it as a literal? Since it's repeating. I would call it by some descriptive name (i.e. validHexStringRegExp) and define as a const variable somewhere at the top.

Reply:
moved the logic into separate function.